caffe2/python/operator_test/cross_entropy_ops_test.py

```diff
@@ -248,6 +250,7 @@ def weighted_sigmoid_xentr_logit_grad_ref(g_out, outputs, fwd_inputs):
             output_to_grad='xentropy',
             grad_reference=weighted_sigmoid_xentr_logit_grad_ref)
 
+    @unittest.skipIf("IN_CIRCLECI" in os.environ, "FIXME: flaky test in CircleCI")
```

Comment:
Why not actually have a real variable for this test? Instead of always testing `os.environ` directly.

Reply:
Noted. Going to do it when we know we are moving forward on the CircleCI path.